README.md

```diff
@@ -11,64 +11,47 @@ pip install grpcio-opentracing
 
 ## Getting started
 
-See the below code for basic usage or [examples/trivial](examples/trivial) for a
-complete example.
+using official grpc interceptor
 
 ### Client-side usage example
 
 ```python
-from grpc_opentracing import open_tracing_client_interceptor
-from grpc_opentracing.grpcext import intercept_channel
+import grpc
+from grpc_opentracing.grpc_interceptor import client_interceptor
 
 tracer = # some OpenTracing Tracer instance
-interceptor = open_tracing_client_interceptor(tracer)
+tracer_interceptor = client_interceptor.OpenTracingClientInterceptor(tracer)
 channel = # the grpc.Channel you created to invoke RPCs
-channel = intercept_channel(channel, interceptor)
+channel = grpc.intercept_channel(channel, tracer_interceptor)
 
 # All future RPC activity involving `channel` will be automatically traced.
 ```
 
 ### Server-side usage example
 
 ```python
-from grpc_opentracing import open_tracing_server_interceptor
-from grpc_opentracing.grpcext import intercept_server
+import grpc
+from concurrent import futures
+from grpc_opentracing.grpc_interceptor import server_interceptor
 
 tracer = # some OpenTracing Tracer instance
-interceptor = open_tracing_server_interceptor(tracer)
-server = # the grpc.Server you created to receive RPCs
-server = intercept_server(server, interceptor)
-
+tracer_interceptor = server_interceptor.OpenTracingServerInterceptor(tracer)
+server = grpc.server(
+    futures.ThreadPoolExecutor(max_workers=10),
+    interceptors=(tracer_interceptor,))
 # All future RPC activity involving `server` will be automatically traced.
 ```
 
 ### Integrating with other spans.
 
-`grpcio-opentracing` provides features that let you connect its span with other
-tracing spans. On the client-side, you can write a class that derives from
-`ActiveSpanSource` and provide it when creating the interceptor.
-
 ```python
-class CustomActiveSpanSource(ActiveSpanSource):
-  @classmethod
-  def get_active_span(self):
-    # your custom method of getting the active span
-tracer = # some OpenTracing Tracer instance
-interceptor = open_tracing_client_interceptor(
-                  tracer,
-                  active_span_source=CustomActiveSpanSource)
-...
-```
-
-On the server-side, the `context` argument passed into your service methods
-packages the gRPC span created on the server-side.
+from grpc_opentracing import scope
```

Comment:
I have mixed feelings about this, as this will probably get discarded anyway in favor of the OT 2.0 API...

Reply:
I looked into current OT  [ScopeManager](https://github.com/opentracing/opentracing-python/blob/master/opentracing/scope_managers/__init__.py#L30) when I wrote this, and thinking it not meet the need. If we can use OT 2.0 API (ScopeManager) would be great.